pages/clustering/high-availability.mdx

```diff
@@ -384,6 +386,19 @@ server that coordinator instances use for communication. Flag `--instance-health
 check the status of the replication instance to update its status. Flag `--instance-down-timeout-sec` gives the user the ability to control how much time should
 pass before the coordinator starts considering the instance to be down. 
 
+There is also a configuration option for specifying whether reads from the main are enabled. The configuration value is by default false but can be changed in run-time 
+using the following query:
+
+```
+SET COORDINATOR SETTING 'enabled_reads_on_main' TO 'true'/'false' ;
+```
+
+All run-time configuration options can be retrieved using:
+
+```
+SHOW COORDINATOR SETTINGS ;
+```
+
```

Comment:
I would consider introducting a subsection for run-time flags here. 

I would name the parent paragraph from `Setting config for highly-available cluster` to -> `Configuring highly-available cluster `

Then that section would have a runtime subsection and a startup configuration.